BadApi.Database/Scripts/PostDeploymentScript.sql

```diff
@@ -0,0 +1,23 @@
+﻿/*
+Post-Deployment Script Template							
+--------------------------------------------------------------------------------------
+ This file contains SQL statements that will be appended to the build script.		
+ Use SQLCMD syntax to include a file in the post-deployment script.			
+ Example:      :r .\myfile.sql								
+ Use SQLCMD syntax to reference a variable in the post-deployment script.		
+ Example:      :setvar TableName MyTable							
+               SELECT * FROM [$(TableName)]					
+--------------------------------------------------------------------------------------
+*/
+MERGE
+	dbo.Account t
+USING
+	(
+		VALUES
+			('user@api.com', 'Password123')
```

Comment:
1.Password seems to be persisted unencrypted. Strongly recommend encryption
2. Consider avoiding hardcoding values into the statement and instead retrieving them from configuration management at run time